ArchiSteamFarm/Storage/GlobalConfig.cs

```diff
@@ -259,9 +256,6 @@ public sealed class GlobalConfig {
 	[JsonProperty(Required = Required.DisallowNull)]
 	public EOptimizationMode OptimizationMode { get; private set; } = DefaultOptimizationMode;
 
-	[JsonProperty(Required = Required.DisallowNull)]
-	public bool Statistics { get; private set; } = DefaultStatistics;
```

Comment:
I'd prefer if you deprecated `Statistics` for one version and (in case of a `false` value) apply the setting to all bot configs after they are loaded. We've automatically migrated previous config property (name) changes and we should do a similar thing here.

Reply:
@Abrynos Feel free to test if it works properly after https://github.com/JustArchiNET/ArchiSteamFarm/pull/2505/commits/451545a09fc437599e06a1954eb6a4926d1fd51c